src/app/news/news-list.component.ts

```diff
@@ -52,6 +55,8 @@ export class NewsListComponent implements OnChanges {
     if (this.replyViewing._id !== 'root') {
       this.replyViewing = this.items.find(item => item._id === this.replyViewing._id);
     }
+    this.usersService.requestUsers();
+    this.activeMembers = this.usersService.data;
```

Comment:
Two things
- `requestUsers` starts a request to the database which will not complete when line 59 runs.  You need to subscribe to `usersListener` to get the updated data based on the database request.
- You should use the array of users.  You don't need `loginActivities` or `childUsers`.

Reply:
@paulbert still unsure where to call `requestUsers`